src/OpenTelemetry.Exporter.Zipkin/Implementation/ZipkinSpan.cs

```diff
@@ -284,15 +285,11 @@ public void Write(Utf8JsonWriter writer)
                 {
                     if (tag.Value is int intValue)
```

Comment:
not from your change but I'm wondering if it would look more concise if using `switch` with type pattern.

Reply:
just changed. Can you check again?